src/neuroconv/nwbconverter.py

```diff
@@ -53,26 +54,23 @@ def get_source_schema(cls) -> dict:
             source_schema["properties"].update({interface_name: unroot_schema(data_interface.get_source_schema())})
         return source_schema
 
-    def get_conversion_options_schema(self) -> dict:
-        """Compile conversion option schemas from each of the data interface classes."""
-        conversion_options_schema = get_base_schema(
-            root=True,
-            id_="conversion_options.schema.json",
-            title="Conversion options schema",
-            description="Schema for the conversion options",
-            version="0.1.0",
-        )
-        for interface_name, data_interface in self.data_interface_objects.items():
-            conversion_options_schema["properties"].update(
-                {interface_name: unroot_schema(data_interface.get_conversion_options_schema())}
-            )
-        return conversion_options_schema
-
     @classmethod
     def validate_source(cls, source_data: Dict[str, dict], verbose: bool = True):
         """Validate source_data against Converter source_schema."""
         cls._validate_source_data(source_data=source_data, verbose=verbose)
 
+    def _validate_source_data(self, source_data: Dict[str, dict], verbose: bool = True):
```

Comment:
@h-mayorquin Just seems like this should be closer to the method above rather than distant from it